ubuntudesign/docker-django-app
# Simplify requirements checking

Change how we check if the requirements have changed, by using a hash of the `requirements.txt` file. This should be more reliable.

Also, merge all the bash code into one file (`entrypoint`). It's a bit easier to track down what's going on that way.

Qa
---

``` bash
git clone git@github.com:canonical-websites/www.ubuntu.com
cd www.ubuntu.com
docker run -ti \
         --volume `pwd`:`pwd` \
         --volume dependencies:/usr/local/lib/python2.7/site-packages \
         --workdir `pwd`  \
         --publish 8000:8000 \
         ubuntudesign/django-app
```

Watch dependencies install. Go to <http://127.0.0.1:8000>, check the site works (styling might be missing).

Now cancel it, try it again:

``` bash
docker run -ti \
         --volume `pwd`:`pwd` \
         --volume dependencies:/usr/local/lib/python2.7/site-packages \
         --workdir `pwd`  \
         --publish 8000:8000 \
         ubuntudesign/django-app
```

This time it shouldn't install dependencies, it should just run the site.
